LICENCE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (C) 2013 Crown Copyright (Government Digital Service)
```

Comment:
Looking back over the commit history it looks like there haven't been significant changes that might qualify for copyright in themselves, they are more keeping this app up to date with other apps, so I'd be inclined to leave it as 2013.

What does it mean that you've approved it? Does that mean I can merge it or are we waiting for someone else to?